tests/quantization/test_precision_init.py

```diff
@@ -119,7 +119,8 @@ def create_hawq_test_config(batch_size=10, num_data_points=100):
 
 def create_hawq_hw_test_config(batch_size):
     config = create_hawq_test_config(batch_size)
-    config["hw_config_type"] = HWConfigType.VPU.value
+    config['quantizer_setup_type'] = "propagation_based"
```

Comment:
Don't we have propagation by default now?

Reply:
Done.